docker/debian/Dockerfile

```diff
@@ -65,15 +65,24 @@ COPY --from=build /usr/local/lib/libprime_server.a /usr/lib/libprime_server.a
 COPY --from=build /home/valhalla/valhalla/build/valhalla-3.2.0-Linux.tar.gz ./
 RUN tar -xzvf valhalla-3.2.0-Linux.tar.gz && cd valhalla-3.2.0-Linux && cp -r bin/* /usr/bin/ && cp -r lib/* /usr/lib/ && cp -r include/* /usr/include/ && cp -r share/* /usr/share/
 
-### Installation des dépendances pour R2GG
+### Installation des dépendances pour psycopg2
+RUN apt update && \
+    apt install -y libpq-dev gcc
```

Comment:
Why gcc ? Did you see an error that was fixed by adding gcc ? why not ;) I'm just trying to understand.

Reply:
This is needed for psycopg2 installation. Otherwise we must use psycopg2-binary which is not recommanded.

Reply:
`gcc` is a requirement used to install `psycopg2` without using `psycopg2-binary` ;-)

You can try to install it (with or without these dependencies) by building this Dockerfile:

```
FROM python:3.6.13-slim-buster as build

RUN apt update && apt install -y libpq-dev

RUN python -m pip install --no-cache-dir -U pip && \
    python -m pip install --no-cache-dir -U setuptools wheel

RUN pip install 'psycopg2==2.8.5'
```

Reply:
ok !